src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx

```diff
@@ -86,7 +97,11 @@ const SaleHeaderBanner = forwardRef<HTMLDivElement, SaleHeaderBannerProps>(
         />
         <PageMargin className={`${baseClassName}__stack-wrapper`} {...commonProps} {...props} ref={ref}>
           <div className={`${baseClassName}__stack`}>
-            {isOpenForBidding && children}
+            {isOpenForBidding && auctionEndTime && (
```

Comment:
apparently we prefer doing this
```
{isOpenForBidding && auctionEndTime ? (<Component />) : null}
```
but I think it's ok if there are no numbers involved
sometimes you get weird stuff where if `auctionEndTIme` was 0, it would just render a 0 on the page
but that shouldn't happen here

Reply:
That's usually how I prefer to do it, but I was just following the convention that was already in the file... but I just pushed up a refactor commit to change all the conditional rendering in this file to ternarys